web/.storybook/preview.js

```diff
@@ -2,6 +2,7 @@ import React from 'react'
 import { ThemeProvider } from '@material-ui/core/styles'
 
 import { theme } from '../src/app/theme'
+import '../src/index.css'
```

Comment:
Double-checking because my memory of this issue is foggy... wherever possible we want absolute filepaths for import statements, but Storybook only accepts relative filepaths. Is that correct?

Reply:
We want to use absolute paths in files under `/src` (I believe it's possible because we set `baseurl` to `src` in `tsconfig.json`.